### PR TITLE
Update mobs/npc template for hib tutorial quest

### PR DESCRIPTION
**Pull request by DigitalBox98**
Update mobs/npc template for hib tutorial quest

### CODE REVIEW

File: data/Mob.0.json [hunk 1]
```diff
@@ -2101450,54 +2101750,54 @@
...
     "RoamingRange": -1,
     "Size": 52,
-    "Speed": 180,
-    "Strength": 0,
-    "Suffix": null,
-    "TranslationId": null,
+    "Speed": 200,
+    "Strength": 110,
+    "Suffix": "",
+    "TranslationId": "",
```
Comment (LRFalk01) [inline data/Mob.0.json:2101796]: I'm seeing a lot of nulls becoming empty strings. Why is this?

Comment (DigitalBox98) [reply data/Mob.0.json:2101796]: Mmm strange indeed I will check again as I didn’t modify this field in my testing DB

Comment (LRFalk01) [reply data/Mob.0.json:2101796]: Did you use DOLToolbox? I wonder if it is doing it.

Comment (DigitalBox98) [reply data/Mob.0.json:2101796]: No I have used /mob create or copy directly in game

Comment (DigitalBox98) [reply data/Mob.0.json:2101796]: Ok I've made a check on the tutorial region for my testing DB : 

In game : 
/mob create DOL.GS.GameNPC

In DB : 
SELECT m.Suffix, m.* FROM mob m WHERE Region=27 AND NAME='New Mob' AND Suffix=''
=> The query is returning the new mob meaning it's the current behavior 

Before my PR : 
SELECT COUNT(1) FROM mob m WHERE Region=27 AND Suffix IS null
=> 2602
SELECT COUNT(1) FROM mob m WHERE Region=27 AND Suffix=''
=> 382

After my PR : 
SELECT COUNT(1) FROM mob m WHERE Region=27 AND Suffix IS null
=> 2601
SELECT COUNT(1) FROM mob m WHERE Region=27 AND Suffix=''
=> 391

So only a portion of Suffix is blank. What do you mean by "a lot of nulls becoming empty strings" ? As I didn't notice a lot of them

Comment (LRFalk01) [reply data/Mob.0.json:2101796]: I think it's fine. I'll approve the PR. I think a followup task for myself could be to convert empty strings to null in the data.

**Review by LRFalk01 — APPROVED** [review]

**Pull request merged**